includes/core/exceptions/class-server-request-exception.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Class Server_Response_Exception
```

Comment:
```suggestion
 * Class Server_Request_Exception
```

Request or response? :P This applies to the class description below as well.